imap/jmap_mail.c

```diff
@@ -12898,6 +12914,9 @@ static int _email_bulkupdate_open(jmap_req_t *req, struct email_bulkupdate *bulk
     }
     strarray_fini(&email_ids);
 
+    /* Always use to account to open destination mailboxes */
+    bulk->req->accountid = accountid;
```

Comment:
This looks dangerous to, this part of the code really shouldn't mess with `req`. It also looks as if this function to reset `req->accountid` to its former value again?